platform/tests/UNITTESTS/doubles/CMakeLists.txt

```diff
@@ -1,8 +1,17 @@
 # Copyright (c) 2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-add_library(mbed-stubs-platform)
+add_library(mbed-headers-platform INTERFACE)
+target_include_directories(mbed-headers-platform
+    INTERFACE
+        ${mbed-os_SOURCE_DIR}/platform/include
+        ${mbed-os_SOURCE_DIR}/platform/include/platform
+        ${mbed-os_SOURCE_DIR}/platform/randlib/include/mbed-client-randlib/
+        ${mbed-os_SOURCE_DIR}/platform/randlib/include/
+        ${mbed-os_SOURCE_DIR}/platform/mbed-trace/include
+)
 
+add_library(mbed-stubs-platform)
```

Comment:
stubs or doubles (the folder is named doubles but we talk or used stubs). Was there a reason to name the folder differently?

Reply:
I thought doubles was more "correct" as some of these are stubs, some are a bit more like fakes. I can revert the folder name back to stubs if we think that's better/less confusing.

Reply:
doubles is fine then.